front/lib/actions/mcp_internal_actions/servers/run_agent.ts

```diff
@@ -124,6 +128,33 @@ function createServer(auth: Authenticator): McpServer {
             return makeMCPToolTextError(errorMessage);
           } else if (event.type === "agent_message_success") {
             break;
+          } else if (event.type === "tool_approve_execution") {
+            if (_meta?.progressToken) {
+              await sendNotification({
+                method: "notifications/progress",
+                params: {
+                  progress: 0,
+                  total: 1,
+                  progressToken: _meta?.progressToken,
```

Comment:
I put it because I need to put something, but there is not really a notion of progress for this?